extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -97,6 +97,22 @@ let BoundPropertyDef;
  */
 let BoundElementDef;
 
+/**
+ * The options bag for binding application.
+ * - skipAmpState: If true, skips <amp-state> elements.
+ * - constrain: If provided, restricts application to children of the provided elements.
+ * - evaluateOnly: If provided, caches the evaluated result on each bound element and skips the actual DOM updates.
+ *
+ * @typedef {Object} ApplyOptionsDef
```

Comment:
I think this would be considered a Record

```suggestion
 * @typedef {Record} ApplyOptionsDef
```

Reply:
Done!